src/main/java/com/maxrt/shoeshop/manager/ShoesManager.java

```diff
@@ -0,0 +1,86 @@
+package com.maxrt.shoeshop.manager;
+
+import java.util.Collections;
+import java.util.stream.Collectors;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.maxrt.shoeshop.shoes.ShoeType;
+import com.maxrt.shoeshop.shoes.Shoes;
+
+public class ShoesManager {
+    private final List<Shoes> shoes = new ArrayList<>();
+
+    public ShoesManager() { }
+
+    public void addShoe(Shoes shoe) {
```

Comment:
Getters&Setters have to be replaced by Lombok annotation

Reply:
Replaced getShoes with @Getter